### DISCUSSION TITLE
How to disable nuxt default scrolling animation behavior?

### DISCUSSION THREAD

**Discussion by cdwmhcc** [nuxt/nuxt#18757] [Questions] [answered]
Nuxt has an animation behavior when clicking on the #xxx anchor, how to disable it

**Comment by TheAlexLichter** [comment, marked as answer]
You can override the [`router.options.ts`](https://github.com/nuxt/framework/blob/main/packages/nuxt/src/pages/runtime/router.options.ts) by placing an own file in the `/app` folder and defining the scroll behavior that way.

**Comment by carbdias** [comment]
Hello, could you provide an example of this?
thank you in advance,
